server/src/main/java/org/elasticsearch/index/reindex/ReindexRequest.java

```diff
@@ -39,7 +44,8 @@
  * of reasons, not least of which that scripts are allowed to change the destination request in drastic ways, including changing the index
  * to which documents are written.
  */
-public class ReindexRequest extends AbstractBulkIndexByScrollRequest<ReindexRequest> implements CompositeIndicesRequest {
+public class ReindexRequest extends AbstractBulkIndexByScrollRequest<ReindexRequest>
+    implements CompositeIndicesRequest, ToXContentObject {
```

Comment:
Could you indent this one more level? I think this kind of indenting makes it look like `implements` is part of the class body.